### PR TITLE
Use run defaults for shell, working-directory for pluto/cbbr/colp builds, use docker images for pluto inputs

### PR DESCRIPTION
**Pull request by fvankrieken**
Little bit of maintenance on workflow files

For cbbr/colp/pluto
- use run defaults
- make sure versions for actions, postgis, ubuntu, etc consistent
- cbbr use docker image for build

Just pluto
- use docker image for input processing, including removing docker calls to geosupport. Need to test all to make sure they work properly

All 3 builds running fine in actions

Pluto inputs a little harder to judge, since two of them "fail" regularly due to missing source data retrieved via ssh. This is a known issue. I wrote a ticket, #47 

[Numbldgs](https://github.com/NYCPlanning/data-engineering/actions/workflows/pluto_input_numbldgs.yml) however is working now

### CODE REVIEW

**Comment by fvankrieken** [comment]
Actually not ready for review - was going to just do pluto but I'll quickly normalize some of the other data products that I'd already moved over to shared utils

**Comment by fvankrieken** [comment]
Leaving ZTL out of this as leaving #43 for the fellows

File: .github/workflows/cbbr_build.yml [hunk 1]
```diff
@@ -15,10 +15,22 @@ on:
...
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./db-cbbr/cbbr_build
+    container:
+      image: nycplanning/build-base:latest
+    env:
```
Comment (fvankrieken) [inline .github/workflows/cbbr_build.yml:25]: So having env at the container level (which I did originally when moving to using containers as that was what most examples looked like) makes variables inaccessible to `use`d actions (see data library action in pluto input ymls below), but declaring them at the job level as we have in the past works fine for both parts of the job in the container and these other calls. So think we should go back to norm of env at job level

**Comment by fvankrieken** [comment]
cbbr has an unrelated issue, going to work on that in a separate branch/pr. This imo is good to go, with caveat that the pluto input jobs that don't have source data are failing and hard to test. But this PR doesn't change that and that should be addressed separately

**Comment by fvankrieken** [comment]
Ah I messed up the rebase. Alright lemme fix that

**Comment by fvankrieken** [comment]
> Ah I messed up the rebase. Alright lemme fix that

Fixed

**Comment by fvankrieken** [comment]
Just merged main in, going to rebuild these three datasets one more time

**Comment by fvankrieken** [comment]
I have certainly violated my own principles for this slightly circuitous PR. But good to go!

File: .github/workflows/cbbr_build.yml [hunk 2]
```diff
@@ -15,10 +15,22 @@ on:
...
+      BUILD_ENGINE: postgresql://postgres:postgres@postgis:5432/postgres
+      EDM_DATA: ${{ secrets.EDM_DATA }}
+      AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
     services:
-      postgres:
-        image: postgis/postgis:12-3.0-alpine
+      postgis:
+        image: postgis/postgis:14-3.3-alpine
```
Comment (damonmcc) [inline .github/workflows/cbbr_build.yml:33]: no downstream changes needed when this service's name changes?

**Review by damonmcc — APPROVED** [review]
lgtm!

File: .github/workflows/cbbr_build.yml (hunk 2 above)
Comment (fvankrieken) [reply .github/workflows/cbbr_build.yml:33]: The build_engine reflects this change

Comment (damonmcc) [reply .github/workflows/cbbr_build.yml:33]: oops sorry didn't see the text change within the line change

**Pull request merged**